dashboard/test/controllers/api/v1/peer_review_submissions_controller_test.rb

```diff
@@ -129,6 +129,128 @@ class Api::V1::PeerReviewSubmissionsControllerTest < ActionController::TestCase
     )
   end
 
+  test 'Peer Review report shows first submission date and latest submission date' do
+    sign_out @plc_reviewer
```

Comment:
I wrote this new test to have no dependencies on the global setup used by the rest of this file - I find it hard to keep track of, and it's nice to know all the necessary setup has been encapsulated here.  The only global dependency is this `sign_out` operation which undoes a setup step all the other tests use.  Maybe in future cleanup we can de-global this.